### PR TITLE
Fix ipv6 connection issues

### PR DESCRIPTION
**Pull request by erikmouw**
This should work around the IPv6 connection issues, much in the same way as it was solved in forecast.solar.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -27,6 +27,7 @@ packages = [
 
 [tool.poetry.dependencies]
 aiohttp = ">=3.0.0"
+aiodns = ">=3.0.0"
```
Comment (klaasnicolaas) [inline pyproject.toml:30]: Looks like you didn't add the package correctly, use: `poetry add aiodns`

File: easyenergy/easyenergy.py [hunk 2]
```diff
@@ -20,6 +22,7 @@
 )
 from .models import Electricity, Gas
 
+API_HOST: str = "mijn.easyenergy.com"
```
Comment (klaasnicolaas) [inline easyenergy/easyenergy.py:25]: Please put this line in a separate `const.py` file

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: easyenergy/easyenergy.py (hunk 2 above)
Comment (erikmouw) [reply easyenergy/easyenergy.py]: Done

**Comment by erikmouw** [comment]
Fix some of the issues, need to look at test coverage

File: easyenergy/const.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+"""Constants for EasyEnergy API client."""
+
```
Comment (klaasnicolaas) [inline easyenergy/const.py:2]: Add `from typing import Final`

File: easyenergy/const.py [hunk 4]
```diff
@@ -0,0 +1,3 @@
...
+API_HOST: str = "mijn.easyenergy.com"
```
Comment (klaasnicolaas) [inline easyenergy/const.py:3]:
```suggestion
API_HOST: Final = "mijn.easyenergy.com"
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: easyenergy/const.py (hunk 3 above)
Comment (erikmouw) [reply easyenergy/const.py:3]: Done

**Comment by erikmouw** [comment]
I need some help with extending the test coverage here, I understand it should test the DNSError path and when there's no DNS result, but I'm a bit lost how to do so.

**Comment by klaasnicolaas** [comment]
Unfortunately I cannot help you with that, it is also a puzzle for me that I am also trying to solve for forecast.solar. You probably need to mock that DNSResolver.

**Comment by erikmouw** [comment]
FYI, testing with hass on a RaspberryPi right now. Removed the `mijn.easyenergy.com` entry from `/etc/hosts` file and restarted hass. No issue anymore, EasyEnergy sensor gets updated.

**Review by klaasnicolaas — APPROVED** [review]

**Comment by klaasnicolaas** [comment]
No problem, solved that for you. I think it would be useful to contact easyEnergy to at least make it known that there is a problem with their IPv6

**Pull request merged**